auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -757,12 +895,26 @@ private Void replaceIdentityPrivileged(final LoadedIdentity newIdentity) throws
             }
         }
 
-        private void writeIdentity(final XMLStreamWriter streamWriter, final LoadedIdentity newIdentity) throws XMLStreamException, InvalidKeySpecException, NoSuchAlgorithmException, CertificateEncodingException {
+        private Version requiredVersion(final LoadedIdentity identityToWrite) {
+            // As new functionality is added we will identify if we need to use a later version
+            // if new functionality is used then use the required schema version otherwise fallback
+            // to an older version.
+
+            // Do we require version 1.1?
+            // return Version.VERSION_1_1;
```

Comment:
Looks like we'd need to require the new version if integrity is enabled (i.e., if publicKey, privateKey aren't null).

Reply:
Changed